services/core-web/src/components/admin/AdminNavigation.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import { Menu } from "antd";
+import { DownOutlined } from "@ant-design/icons";
+import { includes } from "lodash";
+import * as routes from "@/constants/routes";
+
+const propTypes = {
+  activeButton: PropTypes.string.isRequired,
+  openSubMenuKey: PropTypes.arrayOf(PropTypes.string).isRequired,
+};
```

Comment:
Is it possible to rewrite this to be more "typescriptish"? 

const propTypes = {
  activeButton: string,
  openSubMenuKey: Array<string>,
};